source/dub/dub.d

```diff
@@ -657,24 +657,24 @@ class Dub {
 			if (!config.length) config = m_project.getDefaultConfiguration(settings.platform, true);
 		}
 
-		auto generator = createProjectGenerator("build", m_project);
+		auto generator = createProjectGenerator(ide, m_project);
 
 		auto test_config = format("%s-test-%s", m_project.rootPackage.name.replace(".", "-").replace(":", "-"), config);
 
 		BuildSettings lbuildsettings = settings.buildSettings;
 		m_project.addBuildSettings(lbuildsettings, settings, config, null, true);
 
 		if (lbuildsettings.targetType == TargetType.none) {
-			logInfo(`Configuration '%s' has target type "none". Skipping test.`, config);
+			logInfo(`Configuration '%s' has target type "none". Skipping test runner build.`, config);
 			return;
 		}
 
 		if (lbuildsettings.targetType == TargetType.executable && config == "unittest") {
-			logInfo("Running custom 'unittest' configuration.", config);
+			logInfo("Building custom 'unittest' configuration.", config);
```

Comment:
targetType is executable, I think keeping this as `Running` makes sense

Reply:
I was thinking about `dub build ...` printing `Running` to be confusing.